packages/nook/src/components/LibraryView/LibraryView.tsx

```diff
@@ -1,5 +1,98 @@
+"use client";
+
+import React from "react";
+import { View, Text, Loader, Divider, Table, Badge, Alert } from "reshaped"
+import Markdown from 'react-markdown';
+
+import { ComponentDoc, PropItem } from "react-docgen-typescript";
```

Comment:
Have you tried `react-docgen` as well? I think this package is coming from styleguidist while react-docgen is the one under the react namespace on github so curious how different their results are